spectf_cloud/README.md

```diff
@@ -24,7 +24,7 @@ To install the `SpecTf Cloud` python package to use as both:
 - A CLI interface for model training and deployment
 follow the below instructions:
 
-1. [Install GDAL](https://gdal.org/en/stable/download.html)
+1. [Install GDAL](https://gdal.org/en/stable/download.html) (if you have trouble installing, try `conda install -c conda-forge gdal`)
```

Comment:
Add this as a conditional on `conda`:

1. [Install GDAL](https://gdal.org/en/stable/download.html)
    - If you are in a conda environment, you may install locally with `conda install -c conda-forge gdal`